lib/xmerl/src/xmerl_otpsgml.erl

```diff
@@ -95,7 +95,6 @@ convert_tag(dt,Attrs) -> convert_tag(tag,Attrs);
 convert_tag(dd,Attrs) -> convert_tag(item,Attrs);
 convert_tag(ul,Attrs) -> convert_tag(list,Attrs);
 convert_tag(li,Attrs) -> convert_tag(item,Attrs);
-convert_tag(tt,Attrs) -> convert_tag(c,Attrs);
```

Comment:
Here again I think this module is probably dead code, but if it were to be used, it would need to still support `tt`.

Reply:
I'll revert this change.

Reply:
8444a3b613